mobx_codegen/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   build: ^1.1.4
   build_resolvers: ^1.3.2
   meta: ^1.1.0
-  mobx: ^1.1.0
+  mobx: ">=1.1.0 <3.0.0"
```

Comment:
You should specifically target the null-safety release for this

Reply:
done